src/main/java/com/synopsys/integration/detect/lifecycle/boot/product/ProductBoot.java

```diff
@@ -110,34 +110,9 @@ private BlackDuckRunData getBlackDuckRunData(
             
             BlackDuckServicesFactory blackDuckServicesFactory = blackDuckConnectivityResult.getBlackDuckServicesFactory();
             setBlackDuckVersionLevel(blackDuckServicesFactory, blackDuckConnectivityResult);
-            BlackDuckRunData bdRunData = null;
-            
             boolean waitAtScanLevel = shouldWaitAtScanLevel(blackDuckConnectivityResult);
-            
-            if (shouldUsePhoneHome(analyticsConfigurationService, blackDuckServicesFactory.getApiDiscovery(), blackDuckServicesFactory.getBlackDuckApiClient())) {
-                try {
-                    PhoneHomeManager phoneHomeManager = productBootFactory.createPhoneHomeManager(blackDuckServicesFactory,
-                            new PhoneHomeCredentialsFactory().getGa4Credentials());
-                    bdRunData = BlackDuckRunData.online(
-                        blackDuckDecision.scanMode(),
-                        blackDuckServicesFactory,
-                        phoneHomeManager,
-                        blackDuckConnectivityResult,
-                        waitAtScanLevel
-                    );
-                } catch (IntegrationException e) {
-                    logger.debug("Failed to fetch Analytics credentials. Skipping phone home. Exception: " + e.getMessage());
-                } catch (JsonSyntaxException e) {
-                    logger.debug("Analytics credentials file syntax is invalid. Skipping phone home. Exception: " + e.getMessage());
-                } finally {
-                    if (bdRunData == null)
-                        bdRunData = BlackDuckRunData.onlineNoPhoneHome(blackDuckDecision.scanMode(), blackDuckServicesFactory, blackDuckConnectivityResult, waitAtScanLevel);
-                }
```

Comment:
Is there a reason that before the refactoring we had a need for the finally block but now we no longer have it?